### PR TITLE
Promoter: fixing issues after dotorg review

### PR DESCRIPTION
**Pull request by henriqueiamarino**
This issue fixes a warning generated by the automated Dotorg [check](https://themes.trac.wordpress.org/ticket/187829):

**WARNING:** Found a translation function that is missing a text-domain in the file patterns/hidden-no-results.php. Function esc_html_x, with the arguments 'No posts were found.', 'promoter'.
Line  9: <p><?php echo esc_html_x('No posts were found.', 'promoter');?></p>

### CODE REVIEW

File: promoter/patterns/archive.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+<!-- wp:query-pagination-previous {"label":"Prev"} /-->
+
+<!-- wp:query-pagination-numbers /-->
+
+<!-- wp:query-pagination-next {"label":"Next"} /-->
+<!-- /wp:query-pagination -->
+
+<!-- wp:query-no-results {"align":"wide"} -->
+<!-- wp:paragraph {"placeholder":"Add text or blocks that will display when a query returns no results."} -->
+<p><?php esc_html_e('No posts were found.', 'promoter');?></p>
```
Comment (MaggieCabrera) [inline promoter/patterns/archive.php]: this is going to produce the same error again

File: promoter/patterns/index.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+<!-- wp:query-pagination-previous {"label":"Prev"} /-->
+
+<!-- wp:query-pagination-numbers /-->
+
+<!-- wp:query-pagination-next {"label":"Next"} /-->
+<!-- /wp:query-pagination -->
+
+<!-- wp:query-no-results {"align":"wide"} -->
+<!-- wp:paragraph {"placeholder":"Add text or blocks that will display when a query returns no results."} -->
+<p><?php esc_html_e('No posts were found.', 'promoter');?></p>
```
Comment (MaggieCabrera) [inline promoter/patterns/index.php]: and this here too

File: promoter/patterns/search.php [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+<!-- wp:query-pagination-previous {"label":"Prev"} /-->
+
+<!-- wp:query-pagination-numbers /-->
+
+<!-- wp:query-pagination-next {"label":"Next"} /-->
+<!-- /wp:query-pagination -->
+
+<!-- wp:query-no-results {"align":"wide"} -->
+<!-- wp:paragraph {"placeholder":"Add text or blocks that will display when a query returns no results."} -->
+<p><?php esc_html_e('No posts were found.', 'promoter');?></p>
```
Comment (MaggieCabrera) [inline promoter/patterns/search.php]: and here

**Comment by henriqueiamarino** [comment]
Thanks @MaggieCabrera. I've repeated the line fixing it on the three instances you mentioned. Let me know if that solves the problem.

**Review by MaggieCabrera — APPROVED** [review]
👍

**Pull request merged**